backend/service/auth/auth_service.go

```diff
@@ -344,13 +512,13 @@ func (s *Service) RefreshHandler() rest.Handle {
 // AuthDoneHandler handles v1.PathForAuthDone
 func (s *Service) AuthDoneHandler() rest.Handle {
 	return func(w http.ResponseWriter, r *http.Request, _ rest.Params) {
-		code, ok := r.URL.Query()["code"]
-		if !ok || len(code) != 1 || code[0] == "" {
-			marshal.WriteJSON(w, r, httperror.WithInvalidRequest("missing code parameter"))
+		token, ok := r.URL.Query()["token"]
```

Comment:
are you sure the Github is not broken with this change?

Reply:
i see in Google settings we use `response_type: code`
so it should be `code` and not `token` here

Reply:
nah,,, when AuthDoneHandler is called the Authorization Code flow is already passed and code is exchanged with token. Here we simply pass the token to the AuthDoneHandler. That is why you later call it TRUSTY_AUTH_TOKEN and not TRUSTY_AUTH_CODE.